workflow-templates/im-build-npm-package.yml

```
@@ -122,46 +131,51 @@ jobs:
         env:
           NODE_AUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }} # Special per-job token generated by GH for interacting with the repo
 
-      - name: Create a tag for main builds
-        uses: actions/github-script@v4
-        if: ${{ env.IS_MERGE_TO_MAIN }} == 'true'
+      - name: Create GitHub Tag and Release
+        if: needs.examine-triggers.outputs.CREATE_RELEASE == 'true'
+        id: create_release
+        uses: im-open/create-release@v3.0.0
         with:
-          github-token: ${{secrets.GITHUB_TOKEN}}
-          script: |
-            await github.git.createRef({
-              owner: context.repo.owner,
-              repo: context.repo.repo,
-              ref: `refs/tags/${{ steps.version.outputs.NEXT_VERSION }}`,
-              sha: context.sha
-            });
+          github-token: ${{ secrets.GITHUB_TOKEN }} # Special per-job token generated by GH for interacting with the repo
+          delete-existing-release: true # Handy when you hit 're-run jobs' on a workflow run
+          tag-name: ${{ steps.version.outputs.NEXT_VERSION }}
+          prerelease: ${{ env.IS_PRERELEASE }}
+          commitish: ${{ env.REF_TO_BUILD_AND_TAG }}
 
-      - name: Cleanup pre-release package versions
-        if: env.IS_MERGE_TO_MAIN == 'true'
+  cleanup-prereleases:
+    if: always() && github.event.action == 'closed'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Cleanup the branch pre-releases for all closed PRs
+        uses: im-open/delete-prereleases-for-branch@v1.0.2
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          branch-name: ${{ github.head_ref }}
+
+      - name: Cleanup start-project pre-release packages
```

We should have them add their project name, or make the step generically named like this:
```suggestion
      - name: Cleanup pre-release packages
```